### ISSUE TITLE
Request Body Content is Missing for GET Routes

### ISSUE THREAD

**Issue by sauterl** [javalin/javalin-openapi#164]
@dzikoysk with the introduciton of #156 , our [previously working specification](https://github.com/dres-dev/DRES/blob/16bfd035d57e813989759c520d4fb3ad33f98886/doc/oas.json) got transformed to a [new, unusable one](https://github.com/dres-dev/DRES/blob/16bfd035d57e813989759c520d4fb3ad33f98886/doc/oas.json) (unusable = the openapi generator cannot generate client code anymore).

Specifically, the diff is that with the introduction of #156, there will always be a

```json
"requestBody":{
  "required": false
}
```

on each operation.

I might be wrong here, however I have the impression that the `requestBody.required = false` statement for a `GET` request is slightly redundant, isn't it?

However, I'm unsure on how to proceed here. Is there a default for e.g. `GET` requests in the annotation ignored now or did the previous specs work even though they shouldn't have?

To summarise, I have the feeling that there are too many `requestBody` printed in the openapi specs now, as the [documentation](https://swagger.io/docs/specification/describing-request-body/) specifies

>  Request bodies are optional by default. To mark the body as required, use required: true.

Ultimately, I guess, on a `GET` operation must not be a `requestBody.required = false` be set, as the `requestBody.conent` is mandatory.

What are your thoughts?

**Comment by dzikoysk** [comment]
The change of behavior was not intentional and that's definitely a regression. I moved some shared impl to standalone methods, maybe I've missed sth. I'll take a look at it, thanks for reporting this.

**Comment by dzikoysk** [comment]
Yup, the `required` property was added to early, so `requestBody.size() > 0` was always true. It should work properly now :)
